docs/releases/5.0.8.txt

```diff
@@ -28,3 +28,6 @@ Bugfixes
 * Fixed a bug in Django 5.0 that caused a system check crash when
   ``ModelAdmin.date_hierarchy`` was a ``GeneratedField`` with an
   ``output_field`` of ``DateField`` or ``DateTimeField`` (:ticket:`35628`).
+
+* Fixed a crash when saving a model with a ``FileField`` with a ``db_default``
```

Comment:
Perhaps "including" or "defining" to avoid the "with a" repetition?
```suggestion
* Fixed a crash when saving a model with a ``FileField`` including a ``db_default``
```